sass/style.scss

```diff
@@ -48,10 +48,13 @@ a {
             a {
                 color: white;
                 display: inline-block;
-                font-size: 20px;
+                font-size: 2vmin;
```

Comment:
Currently vmin is not supported by IE 9. Since the website still needs to be compatible with IE 9, I cannot except this pull request.